src/ui/App.tsx

```diff
@@ -30,6 +30,7 @@ const App: FC = () => {
           </div>
         </div>
       </div>
+      <div id="modal" />
```

Comment:
Using `modal-root` seems a more appropriate naming, since it will contain one or more modals. You also need to update [this line](https://github.com/okp4/dataverse-portal/pull/434/files#diff-1f2db532dd81fe09dc7bcd1929351ec3d45d81a4105b1c7fc164f5f1714834f4R34) in order to target the right modal parent, the modal root